### PR TITLE
7 add business data

### PR DESCRIPTION
**Pull request by StephenZeiler**
Pull request focused on allowing a user to add their business data and store it in the database. The major changes made in this PR are as follows:

Implementing get, post, patch, put, and delete to the API business controller.
Adding CORS middleware.
Adding API services to the front end to get, post, patch, and delete ("BusinessService.tsx" file).
Implemented functionality that allows a user to add business data and post to the database. This includes using state to update text fields and storing the input to variables as well as a submit button. ("CreateBusiness.tsx" file).

### CODE REVIEW

File: API-LocateContractor/obj/staticwebassets.pack.sentinel [hunk 1]
```diff
@@ -13,3 +13,64 @@
 2.0
 2.0
 2.0
+<<<<<<< HEAD
```
Comment (Krescenskok) [inline API-LocateContractor/obj/staticwebassets.pack.sentinel:16]: You might have missed a merge conflict. These lines should be taken out

File: API-LocateContractor/obj/staticwebassets.pack.sentinel [hunk 2]
```diff
@@ -13,3 +13,64 @@
...
+2.0
+2.0
+2.0
+2.0
+2.0
+2.0
+2.0
+2.0
+=======
+>>>>>>> main
```
Comment (Krescenskok) [inline API-LocateContractor/obj/staticwebassets.pack.sentinel]: Same thing here

File: react-locate-contractor/src/Services/BusinessService.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import axios from 'axios';
+import { business } from "../Pages/BusinessCard";
+import { Business } from "@mui/icons-material";
+
+export const getBusiness = async (searchString: string) => {
+    return await (await axios.get(`http://localhost:5149/Business/email/${searchString}`));
```
Comment (zacoope1) [inline react-locate-contractor/src/Services/BusinessService.tsx]: Typically you would want to abstract out the base url 'http://localhost:5149' to your environment or config.

File: react-locate-contractor/src/Services/BusinessService.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+            businessName: userBusiness.businessName,
+            specialty: userBusiness.specialty,
+            hoursOperation: userBusiness.hoursOperation,
+            emailContact: userBusiness.emailContact,
+            phoneContact: userBusiness.phoneContact,
+            services: userBusiness.services,
+            about: userBusiness.about,
+        })
+    };
+    fetch(`http://localhost:5149/Business/`, requestOptions)
```
Comment (zacoope1) [inline react-locate-contractor/src/Services/BusinessService.tsx]: Same as above (abstract out base url)

File: react-locate-contractor/src/Services/BusinessService.tsx [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+        .then(res => res.json())
+}
+
+export const deleteBusiness = async (searchString: string) => {
+    return await (await axios.delete(`http://localhost:5149/Business/${searchString}`));
+}
+
+export const patchBusiness = async (searchString: string) => {
+    return await (await axios.put(`http://localhost:5149/Business/${searchString}`));
```
Comment (zacoope1) [inline react-locate-contractor/src/Services/BusinessService.tsx]: Same as above (abstract out base url)

**Review by Dwernery — APPROVED** [review]

**Pull request merged**